src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -313,11 +310,9 @@ public void validate(MonitorableJob.Status status) {
             // run validation on feed version
             // FIXME: pass status to validate? Or somehow listen to events?
             status.update("Validating feed...", 33);
-            validationResult = GTFS.validate(feedLoadResult.uniqueIdentifier, DataManager.GTFS_DATA_SOURCE);
+            validationResult = GTFS.validate(feedLoadResult.uniqueIdentifier, DataManager.GTFS_DATA_SOURCE, MTCValidator::new);
```

Comment:
@binh-dam-ibigroup, I think we should only run the MTCValidator if the MTC extension is enabled `isExtensionEnabled("mtc")`. We only want the MTC-specific checks running on MTC instances of Data Tools, not for everyone.